### PR TITLE
Go button not clicked due to not being in view

### PR DESCRIPTION
**Pull request by Achilles-96**
Signed-off-by: Raghuram Vadapalli<raghuram.vadapalli@research.iiit.ac.in>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: test/selenium/PmaSeleniumDbEventsTest.php [hunk 1]
```diff
@@ -66,7 +68,7 @@ public function setUpPage()
      */
     public function tearDown()
     {
-        if ($this->_mysqli != null) {
+        if (!isset($this->_mysqli)) {
```
Comment (nijel) [inline test/selenium/PmaSeleniumDbEventsTest.php]: Shouldn't this be otherwise? We want to execute this quire when the connection has been established.

Comment (Achilles-96) [inline test/selenium/PmaSeleniumDbEventsTest.php]: Yes. That makes sense. I didn't think about that. I just tried to fix the error.In `parent::tearDown()` also, queries are ran only after checking connection is present. Should I change it?

Comment (nijel) [inline test/selenium/PmaSeleniumDbEventsTest.php]: The original code didn't work for you?

File: test/selenium/PmaSeleniumDbEventsTest.php [hunk 2]
```diff
@@ -66,7 +68,7 @@ public function setUpPage()
      */
     public function tearDown()
     {
-        if ($this->_mysqli != null) {
+        if (isset($this->_mysqli)) {
```
Comment (Achilles-96) [inline test/selenium/PmaSeleniumDbEventsTest.php]: The old code was giving me error `Undefined property: PMA_SeleniumDbEventsTest::$_mysqli`

Comment (nijel) [inline test/selenium/PmaSeleniumDbEventsTest.php]: That's defined in PMA_SeleniumBase, there should be no reason for that. Also it's set in setUp method, so you should get failure there as well...

Comment (Achilles-96) [inline test/selenium/PmaSeleniumDbEventsTest.php]: Yes. Even I thought the same thing. I convinced myself that somehow `_mysqli` is being unset before `tearDown` is called. But now I see that event scheduler is not being turned off after the test completes. The problem may be somewhere else.

**Comment by nijel** [comment]
Anyway, I've just merged first commit (53770cf934f32edb8a9b3ba85338e2e6a8c884dd) as it definitely makes sense, the changes in events testing certainly need some more tweaking.

**Comment by Achilles-96** [comment]
It was giving undefined property because it was private. I changed it to protected. Now, it works fine. Any other issues?

**Comment by nijel** [comment]
Good point, merging...

**Pull request merged**